builtin/iter_test.mbt

```diff
@@ -259,6 +259,38 @@ test "range" {
   inspect!((1).until(2, step=0), content="[]")
   inspect!(1.0.until(2.0, step=0), content="[]")
   inspect!(0L.until(10, step=0), content="[]")
+  inspect!(
+    (@int.max_value() - 2).until(@int.max_value() - 1, inclusive=true),
+    content="[2147483645, 2147483646]",
+  )
+  inspect!(
+    (@int.max_value() - 1).until(@int.max_value(), inclusive=true),
+    content="[2147483646, 2147483647]",
+  )
+  inspect!(
+    (@int64.max_value() - 2).until(@int64.max_value() - 1L, inclusive=true),
+    content="[9223372036854775805, 9223372036854775806]",
+  )
+  inspect!(
+    (@int64.max_value() - 1).until(@int64.max_value(), inclusive=true),
+    content="[9223372036854775806, 9223372036854775807]",
+  )
+  inspect!(
```

Comment:
Since the accuracy is not enough, the inclusive param doesn't affect the result here.